docs/CONFIGURATION.md

```diff
@@ -42,3 +42,13 @@ A special feature is that CSS groups can also be used as template variables via
 
 ### Example css group:
 ![Manage Groups: Image 1](https://www.oveleon.de/share/github-assets/contao-component-style-manager/2.0/groups-edit.png)
+
+# Hide group titles
+Since oveleon/contao-component-style-manager version 3.1, the group titles will be shown above the StyleManager widget
```

Comment:
```diff
- Since oveleon/contao-component-style-manager version 3.1, the group titles will be shown above the...
+Since version 3.1, the group titles will be shown above the...
```